examples/bytesFromHex/index.mjs

```diff
@@ -0,0 +1,21 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+const stdlib = loadStdlib(process.env);
+const { ethers } = stdlib;
+
+const b4 = '0x01ffc9a7';
```

Comment:
This is not really the value though... why not the actual byte sequence?

Reply:
By byte sequence do you mean `[1, 255, 201, 167]`, `\x01\xff\xc9\xa7`, or something else?

Reply:
The second

Reply:
javascript turns the hexadecimal characters in the second ones into utf-16 characters and cannot be munged properly, i.e produce 2 bytes instead of 1

Reply:
There has to be a way to represent raw bytes

Reply:
Google says JS defines strings as utf16 so maybe we need the cbr of bytes to always be something else?

Reply:
That's kind of what you did but I think the rule you did is too adhoc